exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/stat/ParquetMetaStatCollector.java

```diff
@@ -71,6 +71,26 @@ public ParquetMetaStatCollector(ParquetTableMetadataBase parquetTableMetadata,
     this.implicitColValues = implicitColValues;
   }
 
+  /**
+   * DRILL_6259 : check if one of the fields involved in the filter is an array.
+   * @param fields columns used in the filter.
+   * @return true if one at least is an array, false otherwise.
+   */
+  public boolean isRepeated(Set<SchemaPath> fields) {
```

Comment:
Where did this come from? I did not do such changes in DRILL-6259.

Reply:
I added this function to solve the issue of DRILL_6259_test_data and comply with your comment : "Please add comment in that place in code where you fixed issue with array, explaining this case. Since you did not add fix in a separate commit, it's hard to tell where you made the change"